### ISSUE TITLE
address accessibility issues

### ISSUE THREAD

**Issue by obfusk** [CatimaLoyalty/Website#31]
We should address accessibility issues flagged by Firefox' built-in tools, like low contrast and lack of image alt text.

**Comment by TheLastProject** [comment]
Please name the exact issues that need addressing, especially because (sadly) not everyone uses Firefox and thus getting started on this issue can be a lot of work for some

**Comment by obfusk** [comment]
I don't have a list I can just copy-paste here. But I intend to either list them here or open a PR to address them. Which is why I assigned this issue to myself :)

**Comment by ngdangtu-vn** [comment]
My advice is don't rely on tech to much as there are diffirent standards and sometime they can be conflict. Also about color contract. If most people don't have any problem then that's ok.

I had a case when chrome acessibility tool told me the text isn't visible enough to see but me and my coworker both think it's fine. Even more funny that after we tried to modify to adapt cr suggestion, it make design look ugly and hard to see. Some explaination can be monitor quality and its config.

About `<img>` tag, not every image need alt. If your img is decoration type you should leave it empty rather try to fill some text in.

One last thing, if you concern about improve website you can use some speedtest sevice like google lighthouse.

(I test you site once it and look pretty ok)

**Comment by obfusk** [comment]
> If most people don't have any problem then that's ok.

I don't think it's okay to ignore people with vision problems when there's an easy way to make things more accessible for them, even if "most people don't have a problem". *I* have vision problems and thought it had too little contrast to read comfortably, and I know people with worse issues. I'd like them to be able to use the website too, without issues. Which is what prompted me to look at the automated tests to have a starting point.

> don't rely on tech to much

Fair enough. It has to be an actual problem, not just checking a box because the computer said so :)

> About <img> tag, not every image need alt. If your img is decoration type you should leave it empty rather try to fill some text in.

True. But e.g. the "Get it on F-Droid" badge really should have alt text.

**Comment by ngdangtu-vn** [comment]
> I don't think it's okay to ignore people with vision problems

Please don't phrase it as ignore. More like you scope down target visitors. Go with something like everyone will likely to make you redesign the whole website every year. For instance, the website is pretty red in hero section and logo. Appearantly, color blind ppl might upset to see the site right?

Just balance it and scope down the right part. Well in your case if text shadow is not enough to enhance your ux, can suggest to make different color background to make the text more po0
